app/federation/server/normalizers/index.js

```diff
@@ -0,0 +1,11 @@
+import message from './message';
+import room from './room';
+import subscription from './subscription';
+import user from './user';
+
+export const normalizers = {
```

Comment:
why not `export * from './message'` or 
`export message`... instead of spread the objects?